app/presenters/sufia/work_show_presenter.rb

```diff
@@ -17,6 +17,20 @@ def tweeter
       end
     end
 
+    def collection_presenter_types
+      [
+        "genericwork",
```

Comment:
Assuming genericwork is not a valid assumption anymore.  There is no longer a GenericWork provided by Sufia.